### ISSUE TITLE
Clicking on notification window in Windows 11 when Nicotine+ is minimized does not maximize the app.

### ISSUE THREAD

**Issue by Faskil** [nicotine-plus/nicotine-plus#3085]
Nicotine+ version:  3.3.4
Operating System/Distribution: Windows 11 Pro 23H2

### Describe the bug

When clicking on the notification popup, nothing happens.

### Expected behavior

When getting a notification from a minimized application in Windows 11, clicking on it will display the application window by default.

### Steps to reproduce the bug

Wait for a notification.
Click on it.
Nothing happens.

### Additional context

Not sure if this is a bug or a lacking feature, but I wanted to raise the issue. 😉

**Comment by slook** [comment]
> Not sure if this is a bug or a lacking feature

It is a bug that the feature is lacking!

**Comment by Faskil** [comment]
I can see that it's been implemented in 3.3.5rc1, thanks!
There's still a small issue though: when clicking on the notification, it should open the app in the foreground, currently on my system it's opening it behind all other opened windows. 😉

**Comment by slook** [comment]
Yes, it still has an issue where the notification popup grabs focus when clicking it, preventing the main window from showing.

This issue hasn't been closed yet for that reason.

**Comment by mathiascode** [comment]
I don't think there's anything we can do here. Windows just refuses to focus the window, and even fails to invoke the callback most times when clicking the notification.

Unless someone has more insight, we'll have to wait for https://gitlab.gnome.org/GNOME/glib/-/work_items/3862.

**Comment by mathiascode** [comment]
Some recent developments: https://gitlab.gnome.org/GNOME/glib/-/merge_requests/5332
